relay-server/src/actors/processor.rs

```diff
@@ -2274,23 +2273,18 @@ mod tests {
             item
         });
 
-        let envelope_response = processor
-            .process(ProcessEnvelope {
-                envelope,
-                project_state: Arc::new(ProjectState::allowed()),
-                sampling_project_state: None,
-                start_time: Instant::now(),
-                scoping: Scoping {
-                    project_key: ProjectKey::parse("a94ae32be2584e0bbd7a4cbb95971fee").unwrap(),
-
-                    organization_id: 1,
-                    project_id: ProjectId::new(1),
-                    key_id: None,
-                },
-            })
-            .unwrap();
+        let new_envelope = relay_test::with_system(move || {
```

Comment:
Tests are largely unaltered and simply updated to match the new signatures. Only exception is this test, which now requires an actix system since it will log outcomes internally. The outcomes do not matter for this test.